src/features/donations/index.tsx

```diff
@@ -55,8 +55,11 @@ function DonationsPopup({ onClose, project }: Props): ReactElement {
         setIsPaymentOptionsLoading(true);
 
         const paymentSetupData = await getRequest(
-          `/app/projects/${project.id}/paymentOptions?country=${country}`,
-          handleError
+          `/app/projects/${project.id}/paymentOptions`,
+          handleError,
+          {
+            country: country,
+          }
```

Comment:
The `getRequest` function accepts 3rd param as redirect and 4th param as query

Reply:
I added '/' now.